packages/ckeditor5-image/src/imagecaption/utils.js

```diff
@@ -25,7 +25,9 @@ export function captionElementCreator( view, placeholderText ) {
 		enablePlaceholder( {
 			view,
 			element: editable,
-			text: placeholderText
+			text: placeholderText,
+			// See: https://github.com/ckeditor/ckeditor5/issues/8689.
```

Comment:
I'm not sure if this comment is needed here.

Reply:
I tried to give better context why we hardcode it up here.